File: lecture_002(Normal)/l_002_lastdigitnumber.cpp
#include<iostream>
using namespace std;
int main(int args, char**argvs) {
    int num;
    cout<<"Enter a number:";
    cin>>num;
    num %= 10;
    cout<<"Last digit number is:"<<endl;
    cout<<num;
    
    return 0;
}

File: lecture_002(Normal)/l_002_lastdigitnumber.java
import java.util.Scanner;
public class l_002_lastdigitnumber {
    public static Scanner scn = new Scanner(System.in);
    public static void main(String[] args) {
        System.out.println("Enter a number:");
        int num = scn.nextInt();
        num %= 10;
        System.out.println("Last digit number:" + num);
    }
}